src/proteus/dswx_hls.py

```diff
@@ -3510,9 +3874,16 @@ class in the browse image. False to display PSW-Agg.
     if output_file:
         logger.info(f'    output multi-band file: {output_file}')
     logger.info(f'    DEM file: {dem_file}')
+    logger.info(f'        description: {dem_file_description}')
     logger.info(f'    Copernicus CGLS land cover 100m file: {landcover_file}')
+    logger.info(f'        description:'
+                f' {landcover_file_description}')
     logger.info(f'    WorldCover 10m file: {worldcover_file}')
```

Comment:
These files are only required if `check_ancillary_inputs_coverage` is True. I'm adding some comments to the runconfig to make it clear.

Reply:
Thank you for the updates to the comments! Those are helpful for one of my other notes.

Here, I was asking something subtly different, perhaps I can try to clarify:

The runconfig's comments simply ask for, "ESA WorldCover map file".
In the code itself (at this line and in other places), the code gets more-specific and says, "WorldCover 10m file".

Would it be possible to harmonize all of these comments, and keep them a consistent e.g. "ESA WorldCover 10m file"? (same for the landcover file)

The resolutions (10m and 100m) are critical for the algorithm to work, so it could be good to note them in the runconfig for the users.